iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -495,6 +492,9 @@ public ModuleRegistrationResponse processRegistrationRequest(ModuleRegistrationR
                 Event.Type.USER_IDENTITY_CHANGE,
                 Event.Type.PRODUCT_ACTION);
 
+        // Specify maximum data age (especially relevant for Event Replays)
+        eventProcessingRegistration.setMaxDataAgeHours(365*24);
```

Comment:
```suggestion
        eventProcessingRegistration.setMaxDataAgeHours(-1);
```
To make it unlimited